### DISCUSSION TITLE
_parser.py:1218: UnknownTimezoneWarning: tzname CEST identified but not understood.  Pass `tzinfos` argument in order to correctly return a timezone-aware datetime.

### DISCUSSION THREAD

**Discussion by seededplyr** [Flexget/Flexget#3070] [Q&A] [answered]
This has been showing up lately when running flexget (not in logs I might add)
`/home/mad/flexget/lib64/python3.6/site-packages/dateutil/parser/_parser.py:1218: UnknownTimezoneWarning: tzname CEST identified but not understood.  Pass `tzinfos` argument in order to correctly return a timezone-aware datetime.  In a future version, this will raise an exception.
  category=UnknownTimezoneWarning)`
  Can't find any mention of it in the forums/issues which makes me quite certain I've screwed something up, anyone have any idea what I haven't done or have done incorrectly that is causing this?

**Comment by ChurdCFG** [comment, marked as answer]
So this is not a permanent fix, but I was able to supress the error in my setup by doing the following:

Edit your site-packages/flexget/plugins/input/rss.py file and change this line:
`feedparser.registerDateHandler(lambda date_string: dateutil.parser.parse(date_string).timetuple())
`
To this:
`feedparser.registerDateHandler(lambda date_string: dateutil.parser.parse(date_string, tzinfos={"CEST": "UTC+2"}).timetuple())`

My guess is since this timezone is "Central European Summer Time", it started for you whenever the clocks switched over (not sure though).  Anyway, hope it helps!

**Reply by seededplyr** [reply]
> 
> 
> So this is not a permanent fix, but I was able to supress the error in my setup by doing the following:
> 
> Edit your site-packages/flexget/plugins/input/rss.py file and change this line:
> `feedparser.registerDateHandler(lambda date_string: dateutil.parser.parse(date_string).timetuple()) `
> To this:
> `feedparser.registerDateHandler(lambda date_string: dateutil.parser.parse(date_string, tzinfos={"CEST": "UTC+2"}).timetuple())`
> 
> My guess is since this timezone is "Central European Summer Time", it started for you whenever the clocks switched over (not sure though). Anyway, hope it helps!

Thank you very much for your reply, I have done as you suggested and the issue is resolved. Out of curiosity, CEST is what I should use even though my region is Asia and my timezone is GMT+8 right? Or should I use something else?

**Reply by ChurdCFG** [reply]
Since the error you posted said CEST specifically, that's the one you should use (If it's the same reason I was getting it, whatever feed/feeds you're parsing in flexget has CEST as a timestamp somewhere).

Glad it worked out!

**Reply by seededplyr** [reply]
> 
> 
> Since the error you posted said CEST specifically, that's the one you should use (If it's the same reason I was getting it, whatever feed/feeds you're parsing in flexget has CEST as a timestamp somewhere).
> 
> Glad it worked out!

ah right, that makes sense, thanks again mate.
